### PR TITLE
VEOSS-1721 | Accessibility audit retest - Usability feedback - Duplic…

### PR DESCRIPTION
**Pull request by Lee-Powell**
…ate Hidden/Visible Text

### CODE REVIEW

**Comment by Lee-Powell** [comment]
retest this please

File: conf/messages.en [hunk 1]
```diff
@@ -192,8 +192,8 @@ addTradingName.title = You have added {0} UK trading names
 addTradingName.heading = You have added {0} UK trading names
 addTradingName.maximumReached = You have added the maximum number of trading names.
 addTradingName.error.required = Select yes if you need to add another UK trading name
-addTradingName.change.hidden = Change {0}
-addTradingName.remove.hidden = Remove {0}
+addTradingName.change.hidden = {0}
+addTradingName.remove.hidden = {0}
```
Comment (hmrcvmu) [inline conf/messages.en:196]: Shouldn't we use the value directly if there's no text?

Comment (hmrcvmu) [reply conf/messages.en:196]: I see, it's being used in `addToList.scala.html`. We could update/rename the `changePrefix` and `removePrefix` parameters to be Long

Comment (hmrcvmu) [reply conf/messages.en:196]: Looks like IOSS also uses `= {0}` - feel free to ignore my comment then

**Review by hmrcvmu — APPROVED** [review]

**Pull request merged**